visivo/commands/dist_phase.py

```diff
@@ -1,50 +1,54 @@
-import importlib.resources as resources
+from visivo.utils import VIEWER_PATH
+from visivo.logging.logger import Logger
+import traceback
 
-VIEWER_PATH = resources.files("visivo") / "viewer"
-
-
-def dist_phase(output_dir, working_dir, dist_dir, default_source, dag_filter, threads):
-    from visivo.commands.run_phase import run_phase
+def dist_phase(output_dir, dist_dir):
     import os
     import json
     import shutil
     from glob import glob
     import datetime
 
-    run_phase(
-        output_dir=output_dir,
-        working_dir=working_dir,
-        default_source=default_source,
-        dag_filter=dag_filter,
-        threads=threads,
-    )
+    Logger.instance().info("Creating distribution for project in folder...")
 
     os.makedirs(f"{dist_dir}/data", exist_ok=True)
-    with open(f"{output_dir}/project.json", "r") as f:
-        project_json = json.load(f)
-    created_at = (datetime.datetime.now().isoformat(),)
-    with open(f"{dist_dir}/data/project.json", "w") as f:
-        f.write(
-            json.dumps(
-                {
+    
+    # Copy dashboard thumbnails if they exist, otherwise create an empty folder
+    try:
+        thumbnail_dir = os.path.join(output_dir, "dashboard-thumbnails")
```

Comment:
Checking that all these new files land in the same place as the URL endpoints you have above?  Otherwise a dist being served will fail.